consensus/integtest/node.go

```diff
@@ -18,7 +18,7 @@ func (n nodes) TotalVotingPower(height types.Height) types.VotingPower {
 	return types.VotingPower(len(n))
 }
 
-func (n nodes) ValidatorVotingPower(addr starknet.Address) types.VotingPower {
+func (n nodes) ValidatorVotingPower(height types.Height, addr starknet.Address) types.VotingPower {
 	return types.VotingPower(1)
 }
```

Comment:
Can voting power change through heights I guess 🤔 .

Since this values are meant to be read only, what do you think of passing them as references

Reply:
height is an uint64, so I don't think we can gain any performance by using reference.
But we can consider changing addr, as it's a felt.